### PR TITLE
BrenKt-SK-997-TRANS-WEB-hosting-change_offer

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.it-it.md [hunk 1]
```diff
@@ -103,30 +88,80 @@ Se non vuoi conservare questi dati, è possibile eliminare il tuo SQL privato pr
...
+> [!warning]
+>
+> Quando decidi di attivare l'opzione Boost, questa resta attiva e fatturata **fino a quando non l'hai disattivata**.
+
+Se l'opzione **Boost** è perfetta per te, ecco le istruzioni per **attivare** o **disattivare** questa opzione sul tuo hosting.
+
+> [!tabs]
+> **Attiva l'opzione Boost**
+>>
+>> Nel riquadro `Informazioni generali` del tuo hosting, clicca sul pulsante`...`{.action} a destra di `Boost` e poi su `Boost` `{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.it-it.md]: Some format errors to fix here

File: pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.it-it.md [hunk 2]
```diff
@@ -25,61 +25,46 @@ Il tuo [Spazio Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&fr
 
 ## In pratica
 
-> [!success]
-> 
-> **Hai bisogno di potenziare temporaneamente il tuo hosting?**
+> [!warning]
 >
-> Con l'[opzione Boost](https://www.ovhcloud.com/it/web-hosting/options/boost/), disponibile sulle nostre offerte *Performance*, potrai far evolvere temporaneamente le risorse del tuo hosting per assorbire un picco improvviso di traffico. Se l'aumento prosegue nel tempo, è possibile migrare verso una soluzione superiore per disporre di maggiori risorse in modo permanente.
+> **Avant** tout changement sur votre abonnement actuel, vérifiez si vous êtes concerné par l'une de ces questions :
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.it-it.md]: This whole warning is not translated

File: pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.es-es.md [hunk 3]
```diff
@@ -89,44 +72,94 @@ Una vez que haya eliminado los datos de sus bases de datos, vuelva a calcular el
...
+
+> [!tabs]
+> **Activar la opción Boost**
+>>
+>> En el cuadro `Información general` de su alojamiento, haga clic en el botón `...`{.action} a la derecha de `Boost` y luego en `Booster mi producto`{.action}.<br><br>
+>> ![boost](images/enable_boost.png){.thumbnail}<br>
+>>
+> **Desactivar la opción Boost**
+>>
+>> En la pestaña `Plus` del alojamiento, haga clic en `"Mejorar mi plan`{.action}".<br>
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.es-es.md]: "" should be withdrawn

File: pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.en-gb.md [hunk 4]
```diff
@@ -85,44 +68,94 @@ If you have deleted data from your databases, you can recalculate the quota used
...
 
+#### Completion
+
+Once you have checked these 6 elements, you can [change your plan](#modify).
+
+### Special cases
+
+#### You have a Start 10M solution <a name="start10m"></a>
+
+If you change your [Start10M](https://docs.ovh.com/gb/en/hosting/activate-start10m/) solution, you will only be offered the [Personal Hosting](https://www.ovhcloud.com/en-gb/web-hosting/personal-offer/) plan. However, once you have switched to the Personal hosting plan, you can migrate it to all of our [web hosting plans](https://www.ovhcloud.com/en-gb/web-hosting/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.en-gb.md]:
```suggestion
If you change your [Start10M](https://docs.ovh.com/gb/en/hosting/activate-start10m/) solution, you will only be offered the [Personal Hosting plan](https://www.ovhcloud.com/en-gb/web-hosting/personal-offer/). However, once you have switched to the Personal hosting plan, you can migrate it to all of our [web hosting plans](https://www.ovhcloud.com/en-gb/web-hosting/).
```

File: pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.en-gb.md [hunk 5]
```diff
@@ -85,44 +68,94 @@ If you have deleted data from your databases, you can recalculate the quota used
...
+
+Please follow [these instructions](#modify) to change your plan.
+
+#### Boost your Performance web hosting plan temporarily <a name="boost"></a>
+
+With the [Boost option](https://www.ovhcloud.com/en-gb/web-hosting/options/boost/), available on our *Performance* plans, you can temporarily increase your hosting system’s CPU and RAM resources to absorb seasonal traffic spikes. If this increase is prolonged over time, you can also [switch to the higher-level Performance plan](#modify) to have these resources permanently.
+
+> [!warning]
+>
+> When you choose to enable the Boost option, the option remains active and charged **until you disable it**.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.en-gb.md]:
```suggestion
> When you choose to enable the Boost option, the option remains active and is billed **until you disable it**.
```

File: pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.en-gb.md [hunk 6]
```diff
@@ -85,44 +68,94 @@ If you have deleted data from your databases, you can recalculate the quota used
...
+#### Billing in case of a change of offer <a name="billing"></a>
+
+When you upgrade your original plan to a higher plan, a *prorata temporis* calculation will be applied until the next renewal date for your original plan.
+This calculation corresponds to the price difference between your initial solution and your new solution.
+
+> **Example:**<br>
+>
+> You subscribed to a [Personal Hosting](https://www.ovhcloud.com/en-gb/web-hosting/personal-offer/) subscription on January 1, 2022.
+>
+> On October 31, 2022, you will upgrade from this **Personal** plan to a subscription on the plan >
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.en-gb.md]: The end of this sentence is missing. Please check also the other translations for the same bug.

File: pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.en-gb.md [hunk 7]
```diff
@@ -61,21 +42,23 @@ In the `Plan` box, click the `...`{.action} button to the right of `Solution`, t
...
 
-> [!primary]
->
-> **Special case**
+### Checking that your hosting plan is compatible with a lower plan <a name="checks"></a>
+
+> [!warning]
 >
-> If you wish to upgrade your [Start10M plan](https://docs.ovh.com/gb/en/hosting/activate-start10m/), only the [Personal plan](https://www.ovhcloud.com/en-gb/web-hosting/personal-offer/) will be available. However, once you have switched to the Personal hosting plan, you can migrate it to all of our [web hosting plans](https://www.ovhcloud.com/en-gb/web-hosting/).
+> You can only change your subscription to a lower range plan if it is the **lower** range plan.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/how_to_upgrade_web_hosting_offer/guide.en-gb.md]:
```suggestion
> You can only change your subscription to a lower range plan if it is the immediate **lower** range plan.
```

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**